### ISSUE TITLE
Is it possible, in principle, to build this with Qt4 (possibly disabling select features)?

### ISSUE THREAD

**Issue by barracuda156** [ylsdamxssjxxdd/eva#17]
To be clear, I do not expect the upstream to do this, I am just interested to get an idea of how reliant is the codebase on functionality which is missing in Qt4 and for which no reasonable substitutes exist.

**Comment by ylsdamxssjxxdd** [comment]
I think it's completely possible, as long as your compiler can support both Qt4 and llama.cpp simultaneously😀

**Comment by barracuda156** [comment]
@ylsdamxssjxxdd Yeah, compiler is no issue, gcc14 builds both and can build Qt4-based apps.
